src/components/Links.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import GITHUB_ICON from '../images/github.png';
```

Comment:
Try to use SVGs for better performance.

Reply:
Fixed by bfa9e2e0ad29e1dc46e377d6f84d711fc940b1bc.